openid-sharedsignals-framework-1_0.md

```diff
@@ -1961,10 +1969,10 @@ that they will receive events related to the subject.
 
 ## Information Harvesting {#management-sec-information-harvesting}
 SETs may contain personally identifiable information (PII) or other non-public
-information about the event transmitter, the subject (of an event in the SET),
+information about the Event Transmitter, the subject (of an event in the SET),
 or the relationship between the two. It is important for Event Transmitters to
 understand what information they are revealing to Event Receivers when
-transmitting events to them, lest the event stream become a vector for
+transmitting events to them, lest the Event Stream become a vector for
```

Comment:
Just "Stream"?

Reply:
This one lets keep as is. I think using just "Stream" is ok if its following "Event Stream".